server/svix-server/src/lib.rs

```diff
@@ -15,6 +18,7 @@ use std::{
 use tower::ServiceBuilder;
 use tower_http::cors::{AllowHeaders, Any, CorsLayer};
 use tower_http::trace::TraceLayer;
+use tracing_subscriber::{prelude::__tracing_subscriber_SubscriberExt, util::SubscriberInitExt};
```

Comment:
```suggestion
use tracing_subscriber::{prelude::*, util::SubscriberInitExt};
```
I think importing this explicitly just looks weird, but anyhow, it wasn't imported before, is it even needed?
Also, are you even sure